src/arrayMethodSort.js

```diff
@@ -3,9 +3,41 @@
 /**
  * Implement method Sort
  */
+const defaultCallback = (a, b) => {
+  const strA = String(a);
+  const strB = String(b);
+
+  if (strA > strB) {
+    return 1;
+  };
+
+  if (strA === strB) {
+    return 0;
+  }
+
+  return -1;
+};
+
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction = defaultCallback) {
+    let count;
+
+    do {
+      count = false;
```

Comment:
You changed variable type but you need also change variable name according new type
[Tips on naming boolean variables](https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig)